### PR TITLE
QA8-1384 add TC-104 & TC-106 for Image suit

### PR DESCRIPTION
**Pull request by MariiaSosedova**
Guys please approve a part of tests for Image

### CODE REVIEW

File: test/extended/image.js [hunk 1]
```diff
@@ -17,5 +21,21 @@ describe('Image field suit', function () {
...
+
+        });
+
+        it('TC-106 Verify that user can upload a PNG file by clicking the Image upload area', function () {
+            uploadImage(images.PNG);
+            let imageDisplayed = $(sel.imageDisplayed);
+            expect(imageDisplayed).toBeDisplayed()
+
+        });
+    });
```
Comment (Alex-Mashchenko) [inline test/extended/image.js:40]: delete this lane [40]

File: test/extended/image.js [hunk 2]
```diff
@@ -17,5 +21,21 @@ describe('Image field suit', function () {
         });
 
     });
+    describe('Positive cases', function () {
+
+        it('TC-104 Verify that user can upload a JPEG file by clicking the Image upload area"', function () {
+            uploadImage(images.default);
+            let imageDisplayed = $(sel.imageDisplayed);
+            expect(imageDisplayed).toBeDisplayed()
+
```
Comment (Alex-Mashchenko) [inline test/extended/image.js:30]: delete this lane [30]

File: test/extended/image.js [hunk 3]
```diff
@@ -17,5 +21,21 @@ describe('Image field suit', function () {
         });
```
Comment (Alex-Mashchenko) [inline test/extended/image.js:22]: delete this lane [22]

**Review by Alex-Mashchenko — CHANGES_REQUESTED** [review]

File: test/extended/image.js (hunk 1 above)
Comment (MariiaSosedova) [reply test/extended/image.js:40]: done

File: test/extended/image.js [hunk 4]
```diff
@@ -17,5 +21,21 @@ describe('Image field suit', function () {
...
+            let imageDisplayed = $(sel.imageDisplayed);
+            expect(imageDisplayed).toBeDisplayed()
+
+        });
+
+        it('TC-106 Verify that user can upload a PNG file by clicking the Image upload area', function () {
+            uploadImage(images.PNG);
+            let imageDisplayed = $(sel.imageDisplayed);
+            expect(imageDisplayed).toBeDisplayed()
+
```
Comment (Alex-Mashchenko) [inline test/extended/image.js:37]: delete this lane [37]

File: test/extended/image.js (hunk 2 above)
Comment (MariiaSosedova) [reply test/extended/image.js:30]: done

File: test/extended/image.js (hunk 3 above)
Comment (MariiaSosedova) [reply test/extended/image.js:22]: done

**Review by Alex-Mashchenko — CHANGES_REQUESTED** [review]

File: test/extended/image.js (hunk 4 above)
Comment (MariiaSosedova) [reply test/extended/image.js:35]: done

File: test/extended/image.js [hunk 5]
```diff
@@ -1,21 +1,38 @@
 import sel from '../../data/selectors';
-// import {name, age, gender, story} from '../../data/testDate';
-// import inputValues4 from '../../helpers/methods';
+import {images} from '../../data/testData';
 import ext from '../../data/expected.json'
```
Comment (Alex-Mashchenko) [inline test/extended/image.js:3]: at the end of your string please use ";"

File: test/extended/image.js [hunk 6]
```diff
@@ -1,21 +1,38 @@
...
+        it('TC-104 Verify that user can upload a JPEG file by clicking the Image upload area"', function () {
+            uploadImage(images.default);
+            let imageDisplayed = $(sel.imageDisplayed);
+            expect(imageDisplayed).toBeDisplayed()
+        });
+
+        it('TC-106 Verify that user can upload a PNG file by clicking the Image upload area', function () {
+            uploadImage(images.PNG);
+            let imageDisplayed = $(sel.imageDisplayed);
+            expect(imageDisplayed).toBeDisplayed()
```
Comment (Alex-Mashchenko) [inline test/extended/image.js:34]: at the end of your string please use ";"

**Review by Alex-Mashchenko — CHANGES_REQUESTED** [review]

File: test/extended/image.js [hunk 7]
```diff
@@ -1,21 +1,37 @@
...
             expect(imageLabelSquare).toEqual(ext.imageLabelSquare);
         });
-
     });
+    describe('Positive cases', function () {
 
+        it('TC-104 Verify that user can upload a JPEG file by clicking the Image upload area"', function () {
+            uploadImage(images.default);
+            let imageDisplayed = $(sel.imageDisplayed);
+            expect(imageDisplayed).toBeDisplayed()
```
Comment (Alex-Mashchenko) [inline test/extended/image.js:28]: at the end of your string please use ";" [28]

**Review by Alex-Mashchenko — CHANGES_REQUESTED** [review]

File: test/extended/image.js (hunk 5 above)
Comment (MariiaSosedova) [reply test/extended/image.js]: done

File: test/extended/image.js (hunk 6 above)
Comment (MariiaSosedova) [reply test/extended/image.js]: done

File: test/extended/image.js (hunk 7 above)
Comment (Alex-Mashchenko) [inline test/extended/image.js]: at the end of your string please use ";" [28]

**Review by Alex-Mashchenko — CHANGES_REQUESTED** [review]

File: test/extended/image.js (hunk 7 above)
Comment (MariiaSosedova) [reply test/extended/image.js]: see my last commit, it's done

**Review by Alex-Mashchenko — APPROVED** [review]
Looks good, all tests pass.
image

**Review by denispalkin — APPROVED** [review]
LGTM. Let's merge!

**Pull request merged**